packages/overlay/src/styles/vaadin-overlay-base-styles.js

```diff
@@ -81,7 +81,7 @@ export const overlayStyles = css`
 
   @media (forced-colors: active) {
     [part='overlay'] {
-      border: 3px solid;
+      border: 3px solid !important;
```

Comment:
Could probably also update `vaadin-notification-card` base styles accordingly.

https://github.com/vaadin/web-components/blob/7fef08bc0022a744be2e65e6073d1c8054b7e87f/packages/notification/src/styles/vaadin-notification-card-base-styles.js#L28-L30

Also, the tooltip overlay base styles likely need to be updated:

https://github.com/vaadin/web-components/blob/7fef08bc0022a744be2e65e6073d1c8054b7e87f/packages/tooltip/src/styles/vaadin-tooltip-overlay-base-styles.js#L54-L56

Reply:
Good point. I’ll take a look.

Reply:
Done